content/en/blog/_posts/2022-07-27-sig-apps-GA-1.25/index.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: blog
+title: "sig-apps features graduating to stable in 1.25"
+date: 2022-07-27
+slug: "sig-apps features graduating to stable in 1.25"
+---
+
+**Authors:** Ravi Gudimetla (Apple), Filip Křepinský (Red Hat), Maciej Szulik (Red Hat)
+
+This blog describes the two features namely `minReadySeconds` for StatefulSets and `maxSurge` for DaemonSets that sig-apps is happy to graduate to stable in 1.25.
+
+Specifying `minReadySeconds` slows down a rollout of a StatefulSet when using a `RollingUpdate` `updateStrategy` by waiting for each pod for a desired time. 
+This time can be used for initializing the pod (e.g. warming up the cache) or as a delay before acknowledging the pod.
+
+`MaxSurge` allows a DaemonSet workload to run multiple instances of the same pod on a node during a rollout when using a `RollingUpdate` `updateStrategy`.
+This helps to minimize the downtime of the DaemonSet to other consumers.
+
+These both features are already available in a Deployment (and other workloads). This graduation helps to align this functionality across the workloads.
```

Comment:
> I'd start with a simple use-case describing the necessity for both of these features and also add d add that this effort is to align higher-level workloads controllers between each other, since Deployments already support both of these features. Only then go into details how this works and how is implemented.

@soltysh would you prefer to have also YAML's of the workloads here? I think that would be too verbose...